lte/gateway/python/magma/pipelined/tests/test_restart_resilience.py

```diff
@@ -165,6 +166,10 @@ def tearDownClass(cls):
         stop_ryu_app_thread(cls.thread)
         BridgeTools.destroy_bridge(cls.BRIDGE)
 
+    def test_detach_default_enforcement_flows(self):
```

Comment:
This function is interpreted as a test (because of the prefix `test_`) but it does not assert anything. Iiuc this is only called in test_with_stateless_enforcement_restart - i.e., it should be in-lined.

Reply:
addressed the review comment.